src/script/DeployLineaStateBridge.s.sol

```diff
@@ -16,15 +16,26 @@ contract DeployLineaStateBridge is Script {
     ///                            CONFIG                           ///
     ///////////////////////////////////////////////////////////////////
     string public root = vm.projectRoot();
-    string public path = "src/script/.deploy-config.json";
+    string public path = "./src/script/config.json";
```

Comment:
This path differs from the previous one (`./config.json`). Are they different files?